### ISSUE TITLE
integration tests are not executed in 1.51.6

### ISSUE THREAD

**Issue by watanabe-kyohei** [inspec/inspec#2556]
### Description

After upgrade inspec, kitchen verify does not execute my integration tests.
inspec shows nothing about test results.

### InSpec and Platform Version

```
$ bundle exec inspec --version
1.51.6
$ ruby --version
ruby 2.4.2p198 (2017-09-14 revision 59899) [x86_64-linux]
$ uname -r
4.9.43-17.38.amzn1.x86_64
```

### Replication Case

using docker(kitchen-docker) as provisioner.
bundle update from 1.51.0 to 1.51.6 and run `bundle exec kitchen test`.

I tried to rebuild my containers(rerun kitchen test), but nothing is changed.
And recovered with downdrading to 1.51.0

followings are outputs of each version.

* 1.51.0
```
$ bundle exec kitchen verify
-----> Starting Kitchen (v1.20.0)
-----> Verifying <default-amazonlinux>...
       Loaded tests from {:path=>".home.k.watanabe.LOCAL<cookbook name>.test.integration.default"}

Profile: tests from {:path=>"/home/k.watanabe/LOCAL/<cookbook name>/test/integration/default"} (tests from {:path=>".home.k.watanabe.LOCAL.<cookbook name>.test.integration.default"})
Version: (not specified)
Target:  ssh://kitchen@localhost:32822


  File /etc/sysconfig/network-scripts/ifcfg-eth0
     ✔  should be file
  ...
Test Summary: 25 successful, 0 failures, 1 skipped
       Finished verifying <default-amazonlinux> (0m18.11s).
```

* 1.51.6
```
$ bundle exec kitchen verify
-----> Starting Kitchen (v1.20.0)
-----> Verifying <default-amazonlinux>...
       Loaded tests from {:path=>".home.k.watanabe.LOCAL.<cookbook name>.test.integration.default"}
       Finished verifying <default-amazonlinux> (0m17.85s).
```

### Possible Solutions

sorry, I have no idea without fix version to 1.51.0

### Stacktrace

inspec finished successfully...

**Comment by jquick** [comment]
@watanabe-kyohei - Thanks for the report. This issue should be fixed in the most recent v1.51.18 release. Please let me know if you still have issues.

**Comment by watanabe-kyohei** [comment]
@jquick 
Thanks for your response.
Excellent!! It works as expected.
